site/content/en/docs/Guides/Backfill/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Backfill"
+linkTitle: "Backfill"
+weight: 5
+description: >
+  Details of Open Match Backfill.
+---
+
+## Background
+
+Some scenarios that could happen when playing multiplayer games include:
+
+* A multiplayer game has a player(s) leave the game. To keep the game interesting and fun, matchmaking will replace the missing player with someone new.
+* A large player count game (e.g.: battleroyal) will create a non-full match and allow players to connect to the game server prior to it being full. Players will wait in a lobby on the game server while new players are backfilled into the game until the server is fully populated.
+* A social or world place in a game allows players to see others in the space, but other players are not strictly required. When players enter the game it is preferred to put players into an existing server, preventing players from joining low population servers. As existing servers reach capacity, the game server management service starts up new empty servers. The new empty servers request players to be backfilled into them, allowing more players to join world servers. Servers may be very long living, as new players replace the leaving players.
+* A multiplayer game has a player(s), leave the game, and makes a request for new players to be backfilled into the game. One of the players in the game invites a friend to join using a social platform. The friend joins the game, and the game server cancels its backfill request.
+
+## Main algorithm
+
+1. Player creates Ticket (`T1`).
+2. Director calls `Backend.FetchMatches` with a MatchProfile.
+3. MMF runs `QueryBackfills` and `QueryTickets` using the MatchProfile. It returns `T1`.
+4. MMF use `T1`, constructs a new Backfill (`B1`*) and returns a `Match` containing ticket `T1` and Backfill `B1`.
+5. Director starts allocating GameServer with Backfill `B1` information.
```

Comment:
```suggestion
5. Director starts allocating GameServer with Backfill `B1.Id` information.
```

Reply:
should be only focused on ID?? if eventually something is added to Backfill, the ID is misleading, in my opinion